packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql

```diff
@@ -205,6 +205,11 @@ select fn_db_add_config_value('HostPackagesUpdateTimeInHours','24','general');
 -- Refresh rate (in hours) for available certification check
 select fn_db_add_config_value('CertificationValidityCheckTimeInHours','24','general');
 
+-- Refresh rate (in hours) for outdated artifacts check
+select fn_db_add_config_value('ArtifactsOutdatedCheckTimeInHours','24','general');
+
+select fn_db_add_config_value('AnsibleRunnerArtifactsLifetimeInDays', '14', 'general');
```

Comment:
Please add this also to engine-config, it might be needed to decrease in some heavily loaded environments